driver/ppm_events_public.h

```diff
@@ -1560,10 +1560,12 @@ enum ppm_event_category {
 	EC_IO_OTHER = 34,/* General I/O that is neither read not write (can be file, socket, IPC...) */
 	EC_WAIT = (1 << 6),	/* General wait (can be file, socket, IPC...) */
 	EC_SCHEDULER = (1 << 7),	/* Scheduler event (e.g. context switch) */
-	EC_INTERNAL = (1 << 8),	/* Internal event generated by the libraries and not by drivers */
+	EC_INTERNAL = (1 << 8),	/* Internal event that shouldn't be shown to the user */
```

Comment:
I am not sure about the duality between internal and generical. I think they're basically the same concept.

Reply:
same answer for `EC_UNKNOWN`

Reply:
Moreover there was this problem in using `EC_INTERNAL` https://github.com/falcosecurity/libs/pull/643#discussion_r984491302

Reply:
Here we need to keep a new flag instead in order to not break the old logic, I've renamed it into `EC_METAEVENT`